salt/modules/aptpkg.py

```diff
@@ -2211,6 +2216,9 @@ def expand_repo_def(**kwargs):
                                                  dist)
 
     source_entry = sourceslist.SourceEntry(repo)
+    for list_args in ('architectures', 'comps'):
+        if list_args in kwargs:
+            kwargs[list_args] = kwargs[list_args].split(',')
```

Comment:
The file order is only relevant for the on-disk file, but not for the behavior of apt. I like to have the order preserved to have
```
deb http://example.com/ stretch main non-free contrib
```
instead of the ordered
```
deb http://example.com/ stretch contrib main non-free
```